### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by ghost** [TrinityNET/CentosAuth#3]
Hello, I'm trying to setup your CentosAuth, and I have some question:
I already host CentosAuth Panel on a iis server with enabled php, I can load the index.php page, but I can't login, I think because I miss the database. Can you share some details about it? I'm not very familiar with php.. Is centosAuth a opensource version of TrinitySeal?With which license do you have release it?(MIT,GPL?ecc) Sorry for my bad english. Regards

**Comment by wnelson03** [comment]
No, CentosAuth is not TrinitySeal. I recommend you use https://github.com/KeyAuth/KeyAuth-Source-Code - both CentosAuth and TrinitySeal have public bypasses to them, you'll only get your software cracked by using them.
